File: examples/src/Rendering-Printing/SupportOfDigitalSignature.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.Rendering_Printing
{
    class SupportOfDigitalSignature
    {

        public static void Run() {

            #ExStart:SupportOfDigitalSignature

            dataDir = RunExamples.GetDataDir_Rendering()
            outPath = RunExamples.OutPath

            with slides.Presentation() as pres:
            {
                
                DigitalSignature signature = new DigitalSignature(dataDir + "testsignature1.pfx", @"testpass1")

                
                signature.Comments = "Aspose.Slides digital signing test."

                
                pres.DigitalSignatures.add(signature)


                pres.save(outPath + "SomePresentationSigned.pptx", slides.export.SaveFormat.PPTX)
            }

            #ExEnd:SupportOfDigitalSignature



        }
    }
}

File: examples/src/presentations/saving/support_of_digital_signature.py
import aspose.slides as slides


def save_add_digital_signature(global_opts):
    with slides.Presentation() as pres:
        signature = slides.DigitalSignature(global_opts.data_dir + "cert.pfx", "testpass1")
        signature.comments = "Aspose.Slides digital signing test."
        pres.digital_signatures.add(signature)
        pres.save(global_opts.out_dir + "save_add_digital_signature_out.pptx", slides.export.SaveFormat.PPTX)
